<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QgsProcessingFieldMapPanelBase</class>
 <widget class="QWidget" name="QgsProcessingFieldMapPanelBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>590</width>
    <height>552</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Fields</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QgsFieldMappingWidget" name="mFieldsView" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="buttonLayout">
       <item>
        <widget class="QToolButton" name="mAddButton">
         <property name="toolTip">
          <string>Add new field</string>
         </property>
         <property name="text">
          <string>add</string>
         </property>
         <property name="icon">
          <iconset resource="../../../images/images.qrc">
           <normaloff>:/images/themes/default/mActionNewAttribute.svg</normaloff>:/images/themes/default/mActionNewAttribute.svg</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="mDeleteButton">
         <property name="toolTip">
          <string>Delete selected field</string>
         </property>
         <property name="text">
          <string>delete</string>
         </property>
         <property name="icon">
          <iconset resource="../../../images/images.qrc">
           <normaloff>:/images/themes/default/mActionDeleteAttribute.svg</normaloff>:/images/themes/default/mActionDeleteAttribute.svg</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="mUpButton">
         <property name="toolTip">
          <string>Move selected field up</string>
         </property>
         <property name="text">
          <string>up</string>
         </property>
         <property name="icon">
          <iconset resource="../../../images/images.qrc">
           <normaloff>:/images/themes/default/mActionArrowUp.svg</normaloff>:/images/themes/default/mActionArrowUp.svg</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="mDownButton">
         <property name="toolTip">
          <string>Move selected field down</string>
         </property>
         <property name="text">
          <string>down</string>
         </property>
         <property name="icon">
          <iconset resource="../../../images/images.qrc">
           <normaloff>:/images/themes/default/mActionArrowDown.svg</normaloff>:/images/themes/default/mActionArrowDown.svg</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="mResetButton">
         <property name="toolTip">
          <string>Reset all fields</string>
         </property>
         <property name="text">
          <string>reset</string>
         </property>
         <property name="icon">
          <iconset resource="../../../images/images.qrc">
           <normaloff>:/images/themes/default/mIconClearText.svg</normaloff>:/images/themes/default/mIconClearText.svg</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="mInvertSelectionButton">
         <property name="toolTip">
          <string>Invert fields selection</string>
         </property>
         <property name="icon">
          <iconset resource="../../../images/images.qrc">
           <normaloff>:/images/themes/default/mActionInvertSelection.svg</normaloff>:/images/themes/default/mActionInvertSelection.svg</iconset>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="loadFromLayerLayout">
     <item>
      <widget class="QLabel" name="loadFromLayerLabel">
       <property name="text">
        <string>Load fields from template layer</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QgsMapLayerComboBox" name="mLayerCombo">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mLoadLayerFieldsButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Load fields from selected template layer</string>
       </property>
       <property name="text">
        <string>Load Fields</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsMapLayerComboBox</class>
   <extends>QComboBox</extends>
   <header>qgsmaplayercombobox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QgsFieldMappingWidget</class>
   <extends>QWidget</extends>
   <header>qgsfieldmappingwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../images/images.qrc"/>
 </resources>
 <connections/>
</ui>
